src/main/java/org/jetbrains/bsp/bazel/server/bsp/impl/BuildServerImpl.java

```diff
@@ -110,4 +110,10 @@ public void onBuildExit() {
     return serverRequestHelpers.executeCommand(
         () -> buildServerService.buildTargetCleanCache(cleanCacheParams));
   }
+
+  @Override
+  public CompletableFuture<Object> workspaceReload() {
+    // TODO
+    return null;
```

Comment:
It's probably best to return a [Completed Future](https://docs.oracle.com/javase/8/docs/api/java/util/concurrent/CompletableFuture.html#completedFuture-U-) here